core/domain/exp_domain.py

```diff
@@ -2873,10 +3772,39 @@ def __init__(self, exploration_id, title, category, objective,
                  exploration_model_created_on,
                  exploration_model_last_updated,
                  first_published_msec):
-        """'ratings' is a dict whose keys are '1', '2', '3', '4', '5' and whose
-        values are nonnegative integers representing frequency counts. Note
-        that the keys need to be strings in order for this dict to be
-        JSON-serializable.
+        """Initializes a ExplorationSummary domain object.
+
+        Args:
+        exploration_id: str. The exploration id.
+        title: str. The exploration title.
+        category: str. The exploration category.
+        objective: str. The exploration objective.
+        language_code: str. The code that represents the exploration language.
+        tags: list(str). List of tags.
+        ratings: dict. Dict whose keys are '1', '2', '3', '4', '5' and whose
+            values are nonnegative integers representing frequency counts. Note
+            that the keys need to be strings in order for this dict to be
+            JSON-serializable.
+        scaled_average_rating: float. The average rating.
+        status: str. The status of the exploration.
+        community_owned: bool. Whether the exploration is community-owned.
+        owner_ids: list(str). List of the user ids who are the owner of
+            this exploration.
+        editor_ids: list(str). List of the user ids of the users who have
+            access to edit this exploration.
+        viewer_ids: lsit(str). List of the user ids of the users who have
+            view this exploration.
+        contributor_ids: list(str). List of the user ids of the user who
+            have contributed to  this exploration.
+        contributors_summary: dict. The summary given by the contributors
```

Comment:
This description is a bit unclear. Can you explain the structure and format of the dict (look at how dicts are documented, with type info, elsewhere in the codebase)? What is a "summary", exactly?